website/pages/docs/platform/k8s/helm/examples/standalone-tls.mdx

```diff
@@ -118,11 +73,9 @@ TMPDIR=/tmp
 
 3. Create the certificate
 
-    1. Create a file `${TMPDIR}/csr.yaml` with the following contents:
+    1. Create a file `${TMPDIR/csr.yaml` with the following contents:
 
-        ```
-        $ export CSR_NAME=vault-csr
-        $ cat <<EOF >${TMPDIR}/csr.yaml
```

Comment:
Any reason these are being reverted?

Reply:
This file is going away because it is being split into the files inside the examples/ subdirectory.

Reply:
Sorry, I'm referring to the lines being removed. Github is presenting it to me as `examples.mdx` being renamed to standalone-tls.mdx, so I'm guessing that's why it looks like I'm commenting on a removed file.

Reply:
Ah, gotcha. Sorry I misunderstood. I've re-copied the example from the current version to get rid of all of these rebasing issues.